jenkins/bootstrap.py

```diff
@@ -832,13 +832,13 @@ def bootstrap(args):
     exc_type = None
 
     try:
+        setup_magic_environment(job)
```

Comment:
This has the unintended side effect of changing the value of cwd and thus a lot of the args, does it not?